collectors/mesos/agent/agent_test.go

```diff
@@ -270,6 +270,37 @@ func TestTransform(t *testing.T) {
 			}
 			result := mac.metricsMessages()
 			So(len(result), ShouldEqual, 1) // one container message
+
+			expected_total_stats := map[string]uint64{
```

Comment:
I usually try to put different values into the fixture for each thing i want to test against.  this prevents a certain type of error where you may accidentally translate one value as a different value.